client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -37,7 +38,17 @@ import './style.scss';
 
 const getExternalUrl = ( mailbox, titanAppsUrlPrefix ) => {
 	if ( isTitanMailAccount( mailbox ) ) {
-		return getTitanEmailUrl( titanAppsUrlPrefix, getEmailAddress( mailbox ), true );
+		const isEmbeddedMailboxEnabled = isEnabled( 'emails/embedded-inbox-testing' );
```

Comment:
Let's ensure the feature flag and the variable name are similar. It's easier that way.